examples/dynamo_integration/scripts/plot_throughput_vs_tsq_per_request.py

```
@@ -0,0 +1,874 @@
+            - llm_call_idx: Index of this LLM call within the request
+            - ttft_ms: Time To First Token for this specific call
+            - tps: Tokens per second for this specific call
+            - itl_ms: Median inter-token latency for this specific call
+            - num_tokens: Number of tokens generated in this call
+    """
+    if not csv_path.exists():
+        return None, None
+    
+    try:
+        df = pd.read_csv(csv_path)
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)